### PR TITLE
Update QD to QL 1

### PR DESCRIPTION
**Pull request by brawner**
Signed-off-by: Stephen Brawner <brawner@gmail.com>

### CODE REVIEW

File: rosidl_typesupport_cpp/QUALITY_DECLARATION.md [hunk 1]
```diff
@@ -117,7 +117,13 @@ Current coverage statistics can be viewed [here](https://ci.ros2.org/job/nightly
 
 ### Performance [4.iv]
 
-Most recent performance test results can be found [here](http://build.ros2.org/view/Rci/job/Rci__benchmark_ubuntu_focal_amd64/BenchmarkTable/).
+`rosidl_typesupport_cpp` follows the recommendations for performance testing of C code in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#performance), and opts to do performance analysis on each release rather than each change.
```
Comment (chapulina) [inline rosidl_typesupport_cpp/QUALITY_DECLARATION.md:120]: C -> C++?

File: rosidl_typesupport_c/QUALITY_DECLARATION.md [hunk 2]
```diff
@@ -2,9 +2,9 @@ This document is a declaration of software quality for the `rosidl_typesupport_c
```
Comment (chapulina) [inline rosidl_typesupport_c/QUALITY_DECLARATION.md:2]: How about making the `linux-arm64_release` -> `linux_release` to this file too?

File: rosidl_typesupport_cpp/QUALITY_DECLARATION.md (hunk 1 above)
Comment (brawner) [reply rosidl_typesupport_cpp/QUALITY_DECLARATION.md]: Done

File: rosidl_typesupport_c/QUALITY_DECLARATION.md (hunk 2 above)
Comment (brawner) [reply rosidl_typesupport_c/QUALITY_DECLARATION.md:2]: Done

**Review by chapulina — APPROVED** [review]

**Pull request merged**